mirascope/rag/types.py

```diff
@@ -7,6 +7,13 @@
 ResponseT = TypeVar("ResponseT", bound=Any)
 
 
+class LogfireModel(BaseModel):
```

Comment:
nit: I don't love this refactor. Since it's only written twice, I might actually prefer to just duplicate so it's clear within each class

Reply:
done